### DISCUSSION TITLE
今天我用腾讯云,阿里云,还有自己的美国杜甫都测试了,全线路无法获取到下载测速.不知道是不是cf改了东西

### DISCUSSION THREAD

**Discussion by hktrace** [XIU2/CloudflareSpeedTest#368] [讨论]
### 问题描述

今天我用腾讯云,阿里云,还有自己的美国杜甫都测试了,全线路无法获取到下载测速.不知道是不是cf改了东西

### 软件版本

2.2.2

**Comment by imcarllee** [comment]
你看前面的issue，有这个问题的解决方法

**Comment by XIU2** [comment]
我现在试了下，没啥问题呀？。。。
你用的是默认下载测速地址，还是其他/自建的？

**Comment by pencilnav** [comment]
-url https://cloudflarest.999857.xyz/100mb.zip
-url https://cloudflarest.999857.xyz/200mb.zip
-url https://cloudflarest.blockly.gq/100mb.zip
-url https://cloudflarest.blockly.gq/200mb.zip
-url https://cloudflarest.gssmc.ga/100mb.zip
-url https://cloudflarest.gssmc.ga/200mb.zip
-url https://cloudflarest.gssmc.gq/100mb.zip
-url https://cloudflarest.gssmc.gq/200mb.zip
-url https://cloudflarest.gssmc.ml/100mb.zip
-url https://cloudflarest.gssmc.ml/200mb.zip
-url https://cloudflarest.gssmc.cf/100mb.zip
-url https://cloudflarest.gssmc.cf/200mb.zip
-url https://cloudflarest.gssmc.tk/100mb.zip
-url https://cloudflarest.gssmc.tk/200mb.zip
由 @PencilNavigator 提供，100MB、200 MB

试下以上地址

**Comment by E021ntox** [comment]


**Reply by XIU2** [reply]
你浏览器能访问这个地址吗？不能的话就换成其他的。

**Reply by XIU2** [reply]
这个地址我用 IPv4 访问会提示 403，用 IPv6 的话可以访问，你是不是用的 IPv6 访问的（系统默认优先 IPv6）？

**Reply by XIU2** [reply]
额，我没注意看你一开始发的内容，你说这个我才发现，你设置了 下载速度下限，那这样的话**是正常的**。
你设置了下载速度下限 为 20MB/s ，那么 CloudflareST 下载测速时，遇到下载速度高于 20MB/s 的进度条才会 +1（即进度条 +1 代表已经找到 1 个满足条件的 IP 了），如果一直遇不到就会一直测速下去（进度条也会一直显示为 0），`下载测速队列：176` 就是总共有 176 个 IP 等待下载测速（下载测速是单线程一个一个来的）

**Reply by XIU2** [reply]
我当然知道，但你要知道 CloudflareST 是一个命令行程序，而不是一个可视化的程序，无法同时动态显示那么多东西。
